test/metabase/models/user_test.clj

```diff
@@ -463,16 +463,16 @@
 
 (deftest delete-pulse-subscriptions-when-archived-test
   (testing "Delete a User's Pulse/Alert/Dashboard Subscription subscriptions when they get archived"
-    (mt/with-temp* [User                  [{user-id :id}]
-                    Pulse                 [{pulse-id :id}]
-                    PulseChannel          [{pulse-channel-id :id} {:pulse_id pulse-id}]
-                    PulseChannelRecipient [_ {:pulse_channel_id pulse-channel-id, :user_id user-id}]]
+    (t2.with-temp/with-temp [User                  {user-id :id}          {}
+                             Pulse                 {pulse-id :id}         {}
+                             PulseChannel          {pulse-channel-id :id} {:pulse_id pulse-id}
+                             PulseChannelRecipient _ {:pulse_channel_id pulse-channel-id, :user_id user-id}]
       (letfn [(subscription-exists? []
                 (t2/exists? PulseChannelRecipient :pulse_channel_id pulse-channel-id, :user_id user-id))]
         (testing "Sanity check: subscription should exist"
           (is (subscription-exists?)))
         (testing "user is updated but not archived: don't delete the subscription"
-          (is (pos? (t2/update! User user-id {:is_active true})))
+          (is (pos? (t2/update! User user-id {:is_active true :first_name "New name"})))
```

Comment:
What's the reason for this change?

Reply:
with the new change, `(t2/update!)` will returns 0 because it detects no change between the original version and the updates, so I added `first_name` to make sure the update will be executed.